packages/components/table/src/filter-panel.vue

```diff
@@ -78,7 +78,8 @@
         @click="showFilterPanel"
       >
         <el-icon>
-          <arrow-up v-if="column.filterOpened" />
+          <component :is="filterIcon" v-if="filterIcon" />
+          <arrow-up v-else-if="column.filterOpened" />
```

Comment:
I think using slots here may be more suitable. This can pass the `column.filterOpened` value, and users can switch icons based on the `column.filterOpened`

Reply:
sounds great.